src/vm/win32threadpool.cpp

```diff
@@ -1815,8 +1815,8 @@ Thread* ThreadpoolMgr::CreateUnimpersonatedThread(LPTHREAD_START_ROUTINE lpStart
         // CreateNewThread takes care of reverting any impersonation - so dont do anything here.
         bOK = pThread->CreateNewThread(0,               // default stack size
                                        lpStartAddress,
-                                       lpArgs           //arguments
-                                       );
+                                       lpArgs,           //arguments
+                                       W("ThreadPool"));
```

Comment:
It maybe useful to call this `.NET Core ThreadPool` - to distinguish it from regular Win32 threadpool.